core/ir/aarch64/codec.txt

```diff
@@ -1119,19 +1119,60 @@ x101101011000000000101xxxxxxxxxx  cls     wx0 : wx5
 0001111011100010010000xxxxxxxxxx     fcvt      s0 : h5
 0001111011100010110000xxxxxxxxxx     fcvt      d0 : h5
 
-# FCVTZU (scalar, integer) FP to GPR reg
+# Floating-point convert (scalar)
+0001111000100100000000xxxxxxxxxx     fcvtas    w0 : s5
+1001111000100100000000xxxxxxxxxx     fcvtas    x0 : s5
+0001111001100100000000xxxxxxxxxx     fcvtas    w0 : d5
+1001111001100100000000xxxxxxxxxx     fcvtas    x0 : d5
+0001111000100000000000xxxxxxxxxx     fcvtns    w0 : s5
+1001111000100000000000xxxxxxxxxx     fcvtns    x0 : s5
+0001111001100000000000xxxxxxxxxx     fcvtns    w0 : d5
+1001111001100000000000xxxxxxxxxx     fcvtns    x0 : d5
+0001111000101000000000xxxxxxxxxx     fcvtps    w0 : s5
+1001111000101000000000xxxxxxxxxx     fcvtps    x0 : s5
+0001111001101000000000xxxxxxxxxx     fcvtps    w0 : d5
+1001111001101000000000xxxxxxxxxx     fcvtps    x0 : d5
+0001111000101001000000xxxxxxxxxx     fcvtpu    w0 : s5
```

Comment:
I verified/tested these by calling them via the test code in suite/tests/api/ir_aarch64.c, if the codec.txt definition was wrong, the tests there would fail to encode/decode the instructions. If you have a more concrete method to prove they match i'd be glad to use it.

Reply:
So the ir_aarch64 test is doing internal-consistency testing, right?  I.e., make sure the encoder and decoder and INSTR_CREATE macros all agree.  But it's not making sure the encodings agree with ground truth?  Some comparison to a third-party/external decoder: e.g., turning the bit patterns into hex and passing to llvm-mc/capstone/binutils or sthg, would be the other type of test to do.  But it looked like there was some of that in the dis-a64 test: or at least it has the objdump output printed there even if it's only compared to the DR output by a human?  Still that's better than nothing until later when we have more automated tests w/ identical-format disasm as noted in the other comment.